14/feoh/date-decorator.py

```diff
@@ -0,0 +1,21 @@
+from functools import wraps
+import datetime
+
+
+def prepend_date(func):
+    '''Decorator that prepends the current datetime.now() string repr and a space.'''
+    @wraps(func)  # preserves function meta data
+    def wrapper(*args, **kwargs):
+        return f"{datetime.datetime.now()} " + func(*args, **kwargs)
```

Comment:
no need for f-string here but I see the point because of the whitespace. But I prefer either everything in the fstrinc or just concatenate `datetime.datetime.now() + " " + func(*args, **kwargs)` or for fun `" ".join([datetime.datetime.now(), func(*args, **kwargs)])`. OK I dont really have anything of value to comment on XD